internal/cmd/local/command_kafka.go

```diff
@@ -0,0 +1,45 @@
+package local
+
+import (
+	"context"
+
+	"github.com/docker/docker/client"
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/internal/pkg/errors"
+)
+
+const (
+	confluentLocalImageName     = "confluentinc/confluent-local:latest"
+	confluentLocalContainerName = "confluent-local"
+	localhostPrefix             = "http://localhost:%s"
+	localhost                   = "localhost"
+	kafkaRESTNotReadySuggestion = "Kafka REST connection may not be ready yet. Rerunning the command is suppposed to solve the issue."
+)
+
+func (c *command) newKafkaCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "kafka",
+		Short: "Manage a single-node instance of Apache Kafka.",
+	}
+
+	cmd.AddCommand(c.newKafkaStartCommand())
+	cmd.AddCommand(c.newKafkaStopCommand())
+	cmd.AddCommand(c.newTopicCommand())
+
+	return cmd
+}
+
+func getShortenedContainerId(id string) string {
+	containerIdShortLength := 10
+	return id[:containerIdShortLength]
```

Comment:
```suggestion
	return id[:10]
```

Reply:
could do this but in earlier comments it was suggested a single `10` is not very clear what it means